Runtime/HairSim.cs

```diff
@@ -29,6 +29,21 @@ public static partial class HairSim
 		static MaterialPropertyBlock s_debugDrawPB;
 
 		static RuntimeFlags s_runtimeFlags;
+		
+		static ComputeBuffer s_DummyComputeBuffer;
+		
+		public static ComputeBuffer dummyComputeBuffer
```

Comment:
We can avoid shoehorning this dummy buffer by transitioning the Hair System to use the modern GraphicsBuffer, for which the CoreUtils offer a dummy buffer. (We could also add a compute dummy to the CoreUtil but I would like to avoid touching Core SRP if possible)